doc/rst/users-guide/locality/compilingAndExecutingMultiLocalePrograms.rst

```diff
@@ -19,6 +19,11 @@ options for communication include:
 * ``gasnet``: specifies that communication should be implemented using
   the open-source GASNet library developed at Berkeley.
 
+* ``ofi``: specifies that communication should be implemented using
+  the open-source libfabric library component of OpenFabrics Interfaces
+  (OFI).   This configuration is preliminary; see :ref:`readme-libfabric` for
+  more information.
+
```

Comment:
I'm a little more ok with this reference since it says "preliminary", but I'd still be open for removing this reference for now.

Reply:
After talking about this offline, I'm fine with leaving this reference in